packages/wpcom-checkout/src/checkout-line-items.tsx

```diff
@@ -709,15 +709,15 @@ function FirstTermDiscountCallout( {
 	}
 
 	if ( isMonthlyProduct( product ) ) {
-		return <DiscountCallout>{ translate( 'Discount for first month' ) }</DiscountCallout>;
+		return <DiscountCallout>{ translate( 'Discount for first month *' ) }</DiscountCallout>;
```

Comment:
I have added the String Freeze label to the PR and a comment to all of the new strings explaining the asterisk. Thank you for bringing this up! I learned a lot about how the translation works when deciphering what you had said haha

Reply:
Nice! Looks good to me. Also, since the actual content of the string is not changing, it might be worth pinging someone in @Automattic/i18n to see if there's a way to add the asterisk to the translations without having to redo them all. I'm not sure but they have many magic powers. 😁

Reply:
Will do!

Reply:
About the asterisks - since the change is pretty small, the translations were automatically marked as 'Fuzzy' which means that the existing translations without the asterisks would be used until they're updated. Our contractors have already updated most of the translations in the [deliverable](https://translate.wordpress.com/deliverables/overview/7320393/).